src/components/archive/archive.js

```diff
@@ -43,14 +43,16 @@ export const Archive = ({
         </div>
         <div className="p-2">
           <button className="w-full" type="button" onClick={onTitleClick}>
-            <p
-              className={`normal-case font-bold m-0 text-sm ${
-                showFullTitle ? "" : "clamp"
-              }`}
+            <a
```

Comment:
I'm not a fan of having the archive title act as a link. If you really need to have a link to the Lanraragi client reader, maybe add it in the archive info modal. Another option is to add a setting that lets the user choose between using this app as a reader or the Lanraragi client. If they choose to use the Lanraragi client. Replace the "Read" button with a link to the Lanraragi client reader.

Reply:
It's a bit redundant indeed. I will revert it.